src/material-examples/cdk-popover-edit-cell-span-vanilla-table/cdk-popover-edit-cell-span-vanilla-table-example.ts

```diff
@@ -0,0 +1,45 @@
+import {Component} from '@angular/core';
+import {NgForm} from '@angular/forms';
+
+export interface Person {
+  id: number;
+  firstName: string;
+  middleName: string;
+  lastName: string;
+}
+
+const PERSON_DATA: Person[] = [
+  {id: 1, firstName: 'Terra', middleName: 'Maduin', lastName: 'Branford'},
+  {id: 2, firstName: 'Locke', middleName: '', lastName: 'Cole'},
+  {id: 3, firstName: 'Celes', middleName: 'Gestahl', lastName: 'Chere'},
+  {id: 4, firstName: 'Edgar', middleName: 'Roni', lastName: 'Figaro'},
+  {id: 5, firstName: 'Sabin', middleName: 'Rene', lastName: 'Figaro'},
+  {id: 6, firstName: 'Clyde', middleName: '"Shadow"', lastName: 'Arrowny'},
+  {id: 7, firstName: 'Setzer', middleName: '', lastName: 'Gabbiani'},
+  {id: 8, firstName: 'Cid', middleName: 'Del Norte', lastName: 'Marquez'},
+  {id: 9, firstName: 'Mog', middleName: '', lastName: 'McMoogle'},
+];
+
+/**
+ * @title CDK Popover Edit spanning 3 columns on an HTML data-table
```

Comment:
This will probably go out of sync if we change the template. It would be better if it didn't mention the amount of columns.

Reply:
I mean that instead of saying "spanning 3 columns" we can say "spanning multiple columns".